datacenter/dtr/2.2/guides/admin/configure/deploy-a-cache.md

```diff
@@ -1,57 +1,168 @@
 ---
-title: Cache Docker images
-description: Learn how to configure DTR to have caching and making pulls faster.
+title: Configure your Docker Content Cache
+description: Learn how to configure DTR to have image caching and pull images faster.
 keywords: docker, registry, dtr, cache
 ---
 
-You can configure DTR to have multiple caches. Users can then configure their
-DTR user accounts to specify which cache to pull from. This way users can
-pull Docker images from a cache that is geographically closer to them or
-that allows them faster downloads.
+## What is Docker Content Cache
 
-## How DTR caches works
+Docker Content Cache is a service that is deployed separately from DTR, and
+can be configured to securely cache images from DTR. Users can configure
+their DTR user accounts to specify which cache to pull from. Then, when a
+user pulls from DTR, they will be redirected to pull from their chosen
+Content Cache. By deploying caches geographically closer to remote offices
```

Comment:
Just to clarify geolocation is not supported by the product.
I can make the suggested change.